Source/Atomic/UI/UISelectItem.h

```diff
@@ -80,6 +80,8 @@ class UISelectItemSource : public Object
 
     void Clear() { items_.Clear(); }
 
+    const String& GetItemStr(int index);
+
```

Comment:
Yes, any function /// comment will automatically be added to C# and TS intellisense by the binding system :)  It is an inherited Urho convention too: https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/Atomic/Graphics/Graphics.h#L121